where-to-travel/src/main/webapp/script.js

```diff
@@ -353,25 +349,26 @@ function populatePlaces(placeArray) {
 /**
  * Helper function that returns the an HTML string representing a place card
  * that can be added to the DOM.
- * @param {string} title the place title
- * @param {string} directionsLink the link to the GMaps directions for this place
- * @param {string} timeStr the amount of time it takes to travel to this place, as a string
+ * @param {Object} place Contains name, lat/lng coordinates, place_id, and travel time to place
+ * @return {String} HTML content to place inside infocard corresponding to place
  */
-function getLocationCardHtml(title, address, directionsLink, timeStr) {
-  const iconId = 'icon' + title;
-  // Link to google search query with name and address of place for user to get more information
-  const titleLink = 'https://www.google.com/search?q=' + encodeURIComponent(title + ' ' + address);
+function getLocationCardHtml(place) {
+  const name = place.name;
```

Comment:
What do you think about inlining all of this instead? Instead of declaring new variables, do ${place.name} in the html string below?

Reply:
I think it might be better practice to use a variable because in the case that we somehow change where we're getting the name from, we only will have to change one line of code instead of all of the places where we use the name. Let me know what you think!

Reply:
That reasoning sounds good to me. Resolving this for now